clients/ui/frontend/src/app/pages/modelCatalog/screens/ModelCatalogPage.tsx

```diff
@@ -91,7 +108,7 @@ const ModelCatalogPage: React.FC<ModelCatalogPageProps> = ({ searchTerm }) => {
           </Bullseye>
         </div>
       )}
-    </>
+    </Gallery>
```

Comment:
You moved the "Load more models" button inside the Gallery, which makes it appear in the grid of cards instead of centered below them as it was before.